tests/main/interfaces-nvidia-tegra-drivers-support/test-snapd-nvidia-drivers-support/bin/check

```diff
@@ -0,0 +1,6 @@
+#!/bin/sh
```

Comment:
Perhaps use the test-snapd-sh test snap pattern and call the two commands in the task.yaml file, then the logic of the test is clearer.